README.md

```diff
@@ -34,6 +34,9 @@ Check out [the docs](http://hexdocs.pm/ex_machina/ExMachina.html) for more detai
 Define factories:
 
 ```elixir
+# You could put this in `test/support/factory.ex` if you use it for tests only,
```

Comment:
Oh yeah, I used this in a Phoenix project – didn't think of the greater world. Sounds like a good change.

Also, I'm not sure about the `lib` recommendation – I just got this trying to deploy my app:

>        ** (CompileError) lib/toolbox/factory.ex:3: module ExMachina.Ecto is not loaded and could not be found

(that's the line `use ExMachina.Ecto, repo: Toolbox.Repo`)

This is likely because my mix.exs says

``` elixir
 {:ex_machina, "~> 0.4", only: [:dev, :test]},
```

So in prod, this file which *is* in `elixirc_paths` does not have its dependency.

`test/support` is cleverly only in the path in tests. I'll probably remove the lib recommendation (I don't use it in dev anymore myself) – it could be solved but it might not be a very common case.